charts/graphql-engine/values.yaml

```diff
@@ -96,6 +96,8 @@ healthChecks:
 serviceAccount:
   enabled: false
   name: ""
```

Comment:
IMO we just set `graphql-engine` directly to this `name` field. Users won't need to explore the helm source to know the default service account name

Reply:
That works! I wasn't sure if there was a preference!

Reply:
@hgiasac was that a request or suggestion? Shall I push that as a change?